nilearn/plotting/displays.py

```diff
@@ -946,6 +1021,73 @@ def draw_cross(self, cut_coords=None, **kwargs):
         # the cuts since we are taking the max along one axis
         pass
 
+    def add_graph(self, adjacency_matrix, node_coords,
+                  node_color=None, node_size=None,
+                  edge_cmap=None,
+                  edge_kwargs=None, node_kwargs=None):
+        """Plot undirected graph on each of the axes
+
+            Parameters
+            ----------
+            adjacency_matrix: numpy array of shape (n, n)
+                represents the edges strengths of the graph. Assumed to be
+                a symmetric matrix.
```

Comment:
> I am actually checking that the matrix is symmetric (and then using the lower
> half) at the moment.

OK. Don't put a tolerance on the check that is smaller than 1e-3 * vmax,
because the eye won't tell a difference anyhow.